.github/workflows/ci.yml

```diff
@@ -16,7 +16,12 @@ jobs:
       - run: cargo check --examples --tests --all-targets
       - run: cargo fmt --all -- --check --files-with-diff
       - run: cargo clippy --all-targets --all-features -- -D warnings
-      - run: cargo test --all-features
+      # runc::tests::test_exec needs $XDG_RUNTIME_DIR to be set
```

Comment:
Shouldn't the test set the environment variable if so?

Reply:
No, usually set by pam_systemd (which is missing in GHA)